docs/samples/blocks/GUI/Window/1.edn

```diff
@@ -0,0 +1,21 @@
+;; usage of parameters :Title, :Width, :Height, :Position, and :Contents
+(defloop main-chain
+  (GFX.MainWindow
+   :Title "Main Window" :Width 800 :Height 600
+   :Contents
+   (-> (GUI.Window
+          :Title "canvas1"        ;; displays title-text on title-bar of window
+          :Width (int 400)              ;; rendered window width in pixels
+          ;; :Width (float 0.25)           ;; rendered window width is 1/4th of container window width
+          :Height (int 300)             ;; rendered window height in pixels
+          ;; :Height (float 0.30)          ;; rendered window width is 1/3rd of container window width
+          :Pos (int2 10 10)       ;; pixel position of top-left corner of window
+          ;; :Pos (float2 0.5 0.5)       ;; top-left corner of window placed in the center of container window
+          :Contents               ;; block code to create and display entitites inside window
```

Comment:
Let's try to align those comments like in the next sample.

Reply:
Fixed.